packages/wdio-browserstack-service/src/instrumentation/funnelInstrumentation.ts

```diff
@@ -151,3 +154,57 @@ function getReferrer(framework?: string) {
     const fullName = framework ? 'WebdriverIO-' + framework : 'WebdriverIO'
     return `${fullName}/${BSTACK_SERVICE_VERSION}`
 }
+
+async function sendTcgDownError(config: BrowserStackConfig) {
+    await fireFunnelTestEvent('SDKTestTcgDownResponse', config)
+}
+
+async function sendInvalidTcgAuthResponse(config: BrowserStackConfig) {
+    await fireFunnelTestEvent('SDKTestInvalidTcgAuthResponseWithUserImpact', config)
+}
+
+async function sendTcgAuthFailure(config: BrowserStackConfig) {
+    await fireFunnelTestEvent('SDKTestTcgAuthFailure', config)
```

Comment:
1. Should we move the enums ("SDKTestInvalidTcgAuthResponseWithUserImpact" .. etc) to constants? 
2. Should these events be fire and forget ?

Reply:
1. We can move them to constants if required but the other event names have been hard coded. Since they are only used at one place, i dont think there's a need to move them to constants.
2. These are fire and forget at the place where `sendTcgAuthFailure` function is called. The code structure has been kept consistent with other events.